src/uct/ib/rdmacm/rdmacm_listener.c

```diff
@@ -41,8 +60,8 @@ UCS_CLASS_INIT_FUNC(uct_rdmacm_listener_t, uct_cm_h cm,
         goto err_destroy_id;
     }
 
-    backlog = (params->field_mask & UCT_LISTENER_PARAM_FIELD_BACKLOG) ?
-              params->backlog : SOMAXCONN;
+    backlog = uct_cm_set_listener_backlog(params, ucs_rdmacm_max_backlog());
```

Comment:
it should be out parameter of `uct_listener_query`

Reply:
@yosefe should we add the backlog to uct_listener_attr's api?

Reply:
i don't think we need in this PR, it's just internal value.